### ISSUE TITLE
Heroku breaks widgets-section list at http://www.aftenposten.no

### ISSUE THREAD

**Issue by xbcd** [EFForg/https-everywhere#3041]
At non-HTTPS aftenposten.no pages, e.g.
http://www.aftenposten.no/meninger/sid/Pa-tide-a-legalisere-cannabis-8122474.html
when Heroku rules are **dis**abled in the HTTPS Everywhere settings, there will be 5 files loaded:

http://widget-user-header.herokuapp.com/public/all.js
http://widget-user-header.herokuapp.com/public/all.css
http://widget-list.herokuapp.com/public/section-list.css
http://follow-author.herokuapp.com/tagScripts.js
http://follow-author.herokuapp.com/style.css

These will create a "Widgets-section list" at the bottom of the page:
![Imgur Heroku rules disabled](http://i.imgur.com/I2OXnaV.png)

If the Heroku rules are **en**abled in the HTTPS Everywhere settings (the default)
those 5 files will not be loaded. 
(Even though they may be **manually** downloaded by entering e.g.
https://widget-user-header.herokuapp.com/public/all.js
I don't know the reason why they're not loaded..)

And therefore, that "Widgets-section list" at the bottom of the page will be missing:
![Imgur Heroku rules enabled](http://i.imgur.com/EyapiLK.png)

However, when using HTTP**S** aftenposten.no pages, e.g.
https://www.aftenposten.no/meninger/sid/Pa-tide-a-legalisere-cannabis-8122474.html
then those 5 files will always be loaded as HTTPS, 
no matter if the Heroku rules are enabled or disabled in the HTTPS Everywhere settings.

I see there are HTTPS-E rules for aftenposten.no, but it seems they are disabled?
I will only get HTTP aftenposten.no unless I manually change it to HTTPS.

To summarize, the current workaround is either disabling the Heroku rules,
or else manually change into https://www.aftenposten.no

**Comment by numismatika** [comment]
It is a CORS issue:

``Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at http://widget-list.herokuapp.com/?source=%2Farticle-list%2Fescenic%2Fap%2Fsection%2Fmeninger_sid%3FonBehalfOf%3Dap%26limit%3D6&template=section-list. (Reason: CORS header 'Access-Control-Allow-Origin' missing).
``
you cannot load the herokuapp content  (which is secured) from an insecure website.
There is a file in the addon for this website, but a lot of stuff was excluded from a rewrite to avoid mixed-content breakage.

**Comment by gloomy-ghost** [comment]
massive close per #15157
